steve.gd

```diff
@@ -64,9 +65,10 @@ var aPressed = false
 var bPressed = false
 
```

Comment:
Not sure why this function is called fakeprocess, maybe a better name would be inputloop or something